evennia/commands/default/system.py

```diff
@@ -1203,3 +1205,183 @@ def func(self):
                 "*" if sub[5] else "-",
             )
         self.caller.msg("|wActive tickers|n:\n" + str(table))
+
+
+class CmdTasks(COMMAND_DEFAULT_CLASS):
+    """
+    Manage active tasks (delays).
+
+    Note:
```

Comment:
This note should mention what a 'task' is and that it can only be started from code. Other commands like this have in-game creation equivalents (`scripts` have `script`, `objects` have `create/dig` etc), so reading this one would expect the same for this when reading the help string.

Reply:
Put in a reference in notes and make the header sentence more clear.

I don't see task creation at command line as a value add. Admittingly, I see script creation at command line as a nightmare. So I don't have a very robust grasp of how other people like to manage an environment.